### DISCUSSION TITLE
Unable to get a stable connection

### DISCUSSION THREAD

**Discussion by EuleMitKeule** [dbuezas/eq3btsmart#76] [General]
I am currently using D1 Mini ESP32 bluetooth proxies to connect my EQ-3 valves. Unfortunately, the entities go unavailable very frequently, no matter how close the esp is to the valve. 

I also tried to connect the valves using my mainboard integrated bluetooth and with a bluetooth dongle. With both of those I am able to see the valves using `hcitool lescan` in the terminal. The autodiscovery in Home Assistant also seems to work, but when I try to add the devices, all the entites are unavailable and stay like that.

I would love to help getting this to work, since multiple people encountered stability issues using the bluetooth proxy method.
When using the official Android app the valves are super responsive and there are no connection issues, so I believe this should be totally doable to get a similar experience.

If I can help in any way, e.g. testing out stuff with the bluetooth proxies or working on the code myself, please let me know. For the latter I would appreciate any input or ideas you might have on what could be causing the stability issues. You can also hit me up on Discord @EuleMitKeule.

**Comment by dbuezas** [comment]
Hi!
Let's gather this up together with #75
